lib/core/seleniumRunner.js

```diff
@@ -373,14 +384,16 @@ class SeleniumRunner {
         try {
           return this.driver.executeAsyncScript(script, args);
         } catch (e) {
-          log.error('Couldn\'t execute async script named ' + name + ' error:' + e);
+          log.error(
```

Comment:
All of these whitespace changes will make this harder to upstream, and harder to merge upstream into our code (which I am doing right now).

Are they generated by your IDE?  Try to figure out how to stop that if so.

Reply:
They are actually generated by *your* configuration -- I just ran it through eslint using the configuration that the browsertime repo specifies :-) I was absolutely not trying to change anything. I was just trying to follow the rules.

Reply:
> They are actually generated by _your_ configuration -- I just ran it through eslint using the configuration that the browsertime repo specifies :-) I was absolutely not trying to change anything. I was just trying to follow the rules.

Well, _mine_ in the sense of "whatever was in the browsertime repo".  It's a good idea to run things through linters, etc before landing -- but upstream/downstream trumps whitespace churn.

Is it possible to apply ESlint only to changed lines?  That's best possible here.